styleAbout.css

```diff
@@ -0,0 +1,446 @@
+  color: white;
+}
+
+/* Partners */
+
+.partners {
+  display: block;
+  background-color: var(--main-color);
+  padding: 2rem 1rem;
+}
+
+.partners h2 {
+  color: var(--secondary-color);
+}
+
+.partner {
+  width: 20vw;
+  margin: 1rem;
+}
+
+.partner img {
+  width: 100%;
+}
+
+.festival-partner {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+}
+
+.last-section {
+  display: flex;
+  align-items: center;
+  padding: 2rem 1rem;
+  gap: 2rem;
+  background: rgb(240, 240, 240);
+}
+
+.last-logo {
+  width: 50%;
+  display: flex;
+}
+
+.last-logo img {
+  width: 100%;
+}
+
+.last-text p {
+  font-size: 1rem;
+  font-weight: 500;
+}
+
+@media screen and (min-width: 768px) {
+  #logout {
+    background: var(--main-color);
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    padding: 0.8rem 4rem;
+    gap: 1rem;
+  }
+
+  .logout-link {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+  }
+
+  .logout-link a {
+    color: var(--secondary-color);
+    font-weight: bolder;
+  }
+
+  /* NAVIGATION BAR */
+
+  .logo {
+    display: block;
+    width: 6rem;
+  }
+
+  .logo img {
+    width: 100%;
+  }
+
+  .close {
+    display: none;
+  }
+
+  .nav-bar {
+    background-color: white;
+    padding: 1rem 5rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .nav-links {
+    display: flex;
+    background: none;
+    backdrop-filter: blur(0);
+    margin-left: 0;
+    width: fit-content;
+    height: max-content;
+    top: inherit;
+    left: inherit;
+    position: inherit;
+  }
+
+  .nav-links ul {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 1.5rem;
+  }
+
+  .nav-links a {
+    color: var(--main-color);
+    font-size: 1rem;
+    font-weight: 500;
+  }
+
+  .initial a {
+    color: var(--primary-color);
+  }
+
+  .campaign {
+    border: 5px solid #ec5242;
+    padding: 0.6rem 1rem;
+  }
+
+  .campaign a {
+    color: var(--primary-color);
+  }
+
+  /* INTRODUCTION */
+
+  .introduction {
+    padding: 10rem 8rem;
+    gap: 3rem;
+  }
+
+  .introduction h1 {
+    font-size: 3.5rem;
+    width: 60vw;
+    margin: 0 auto;
+  }
+
+  .intro-1 {
+    display: none;
+  }
+
+  .intro-2 {
+    text-align: center;
+    width: 60vw;
+    margin: 0 auto;
+    line-height: 1.5;
+  }
+
+  .contact-us {
+    font-size: 1.4rem;
+  }
+
+  .created-logo {
+    width: 25vw;
+    box-shadow: 0 0 15px #d3d3d3;
+  }
+
+  .past-festival {
+    padding: 2rem 10rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    box-shadow: 0 0 10px #d3d3d3;
+  }
+
+  .last-images {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  }
+
+  .partners {
+    display: none;
+  }
+
+  .last-section {
+    padding: 2rem 10rem;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] In order to follow the design and also replicate what we have in the design perfectly well, I would kindly advise you to give this part of your styling a `background-color: #272a31;` and `color: #fff;`